### PR TITLE
initial benchmarks to try verifying O(2^k * n) for graph coloring

### PR DESCRIPTION
**Pull request by andersenwall**
initial benchmarks complete for constant 2 bags with n nodes. 
verified n linear for false returns on coloring tests
problem found with n worse than exponential for true returns on coloring tests

### CODE REVIEW

File: src/DecidingSheaves.jl [hunk 1]
```diff
@@ -22,51 +22,98 @@ OUTPUT: a structured decomposition obtained by replacing the span de in d
...
   end
-  # d_csp is the cospan dx₁ -> de <- dx₂ corresp to some edge e = x₁x₂ in shape(d)
-  (csp, d_csp)      = tup  #unpack the tuple
-  # the pullback cone dx₁ <-l₁-- p --l₂ --> dx₂ with legs l₁ and l₂
-  p_cone            = pullback(d_csp)
-  p_legs            = legs(p_cone)
-  #for each leg lᵢ : p → xᵢ of the pullback cone, 
-  #compute its image ιᵢ : im lᵢ → dxᵢ
-  imgs              = map( f -> legs(image(f))[1], p_legs)
-  #now get the new desired cospan;
```
Comment (jpfairbanks) [inline src/DecidingSheaves.jl:36]: make sure these comments end up back in the code.

**Comment by jpfairbanks** [comment]
@andersenwall, you have permission from @samuelsonric to replace the failing test with a check that the lengths are equal

**Comment by andersenwall** [comment]
> @andersenwall, you have permission from @samuelsonric to replace the failing test with a check that the lengths are equal

resolved, test cases should all pass now

File: src/DecidingSheaves.jl (hunk 1 above)
Comment (andersenwall) [reply src/DecidingSheaves.jl:36]: comments are back in the code
old versions of the functions are still in the DecidingSheaves file under old_(function name)

**Review by samuelsonric — APPROVED** [review]

**Pull request merged**